### DISCUSSION TITLE
Option to set default nextcloud local / language / default phone region

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#736] [Ideas] [answered]
It would be nice if it could be possible to set the [default location / language](https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/language_configuration.html) and [the default phone region](https://github.com/nextcloud/all-in-one#how-to-resolve-security--setup-warnings-displays-the-missing-default-phone-region-after-initial-install) directly from the aio panel, since these are settings which many would need to set and this would fit good to the timezone settings of the containers. Because the options would need to be set by many people, I think such an option would also be better for persons who don't like command shell. (I hope my Ideas / Requests / discussions aren't too much)

**Comment by szaimen** [comment]
Hello, honestly I think it is not that important to get a separate input field for this. Also since it is a basic occ command you can e.g. also use the occ web app to set it. 

Rather should Nextcloud itself provide an inout fiels for this, imho.

**Reply by Zoey2936** [reply]
> Rather should Nextcloud itself provide an inout fiels for this, imho.

That's right

**Reply by jameskimmel** [reply]
You can't set it for phone in the webGUI.

And as a Docker newcomer and noobie, isn't to whole point of compose that I have a config file I can spin up everywhere, without having to input some manual PHP commands afterwards?

**Comment by szaimen** [comment, marked as answer]
Hi, please create a feature request for this here if not already done: https://github.com/nextcloud/server/issues
